src/safeposix/net.rs

```diff
@@ -313,9 +365,9 @@ impl NetMetadata {
                         if portuser.0 == muxed.0 {
                             //if it's rebindable and we're removing the last bound port or it's just not rebindable
                             if portuser.1 <= 1 {
-                                drop(portuser);
+                                let _ = portuser;
                                 if userarr.len() == 1 {
-                                    drop(userarr);
+                                    let _ = userarr;
```

Comment:
Was there a reason to replace the use of destructor `drop`?

Reply:
The compiler was throwing warnings at these lines and suggested assigning it the unused varuable to  _ instead of dropping it.